### PR TITLE
fix IE 10 issue by adding date_time_format_timezone

### PR DESCRIPTION
**Pull request by skaymen**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: wdfn-server/waterdata/templates/base_plain.html [hunk 1]
```diff
@@ -20,10 +20,19 @@
...
-                'WEATHER_SERVICE_ROOT': '{{ config.WEATHER_SERVICE_ROOT }}'
+                'WEATHER_SERVICE_ROOT': '{{ config.WEATHER_SERVICE_ROOT }}',
+                'DATE_TIME_FORMAT_TIMEZONE': "{{ 'scripts/date-time-format-timezone-complete-min.js' | asset_url }}"
             };
         </script>
         <script src="https://cdn.polyfill.io/v2/polyfill.min.js"></script>
+        <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=Intl.~locale.en"></script>
+        <script type="text/javascript">
+            if (!!document.documentMode) {
+                document.write("<script src=" + ${CONFIG.DATE_TIME_FORMAT_TIMEZONE} + "><\/script>");
```
Comment (mbucknell) [inline wdfn-server/waterdata/templates/base_plain.html]: This throws an error in the browser. For basic string concatentation you don't surround the variable by ${...}, just specify the variable. The other syntax is used for template literals which you can't use directly in the browser anyways.

Comment (danielnaab) [reply wdfn-server/waterdata/templates/base_plain.html]: Why does `DATE_TIME_FORMAT_TIMEZONE` need to be on the config object? Could the script tag just directly reference `{{ 'scripts/date-time-format-timezone-complete-min.js' | asset_url }}`?

Comment (mbucknell) [reply wdfn-server/waterdata/templates/base_plain.html]: I don't think so. The javascript executes the string concatentation so would treat that as a straight string. @skaymen can comment, but I believe he tried it that way initially.

Comment (danielnaab) [reply wdfn-server/waterdata/templates/base_plain.html]: Right, but I mean skip the string concatenation altogether and just render with Jinja:

```
    document.write("<script src="{{ 'scripts/date-time-format-timezone-complete-min.js' | asset_url }}"><\/script>");
```

Comment (mbucknell) [reply wdfn-server/waterdata/templates/base_plain.html]: @danielnaab you are right. @skaymen, try the following: 
`document.write('<script src="{{ "scripts/date-time-format-timezone-complete-min.js" | asset_url }}"' + '<\/script>');`
Then you won't need the variable.

File: assets/package.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
     "watch:css": "nodemon -w src/styles -e scss -x \"npm run build:css\"",
     "watch:fonts": "nodemon -w node_modules/uswds/src/fonts -x \"npm run build:fonts\"",
     "watch:images": "nodemon -w src/img -x \"npm run build:images\"",
-    "watch:js": "rollup -c --watch"
+    "watch:js": "rollup -c --watch && nodemon -w node_modules/date-time-format-timezone/build/browserified/date-time-format-timezone-complete-min.js -x \"npm run build:scripts\" "
```
Comment (mbucknell) [inline assets/package.json]: This is not working. You should just copy the part in line 18 that you added to line 28.

File: assets/package.json [hunk 3]
```diff
@@ -25,7 +25,7 @@
     "watch:css": "nodemon -w src/styles -e scss -x \"npm run build:css\"",
     "watch:fonts": "nodemon -w node_modules/uswds/src/fonts -x \"npm run build:fonts\"",
     "watch:images": "nodemon -w src/img -x \"npm run build:images\"",
-    "watch:js": "rollup -c --watch"
+    "watch:js": "mkdir -p dist/scripts && cp node_modules/date-time-format-timezone/build/browserified/date-time-format-timezone-complete-min.js dist/scripts && rollup -c --watch"
```
Comment (mbucknell) [inline assets/package.json]: You should not have removed the directory creation and cp from the build:js. Please restore it. This is necessary so when the artifiact is build it contains this file. The watch script is not run during the build process on Jenkins. It is strictly for development.

**Pull request merged**